.travis.yml

```diff
@@ -123,7 +123,8 @@ before_install:
     - ${KLEE_SRC}/.travis/install-llvm-and-runtime-compiler.sh
     - ${KLEE_SRC}/.travis/install-tcmalloc.sh
     # Install lit (llvm-lit is not available)
-    - sudo pip install lit
+    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then sudo pip2 install lit; fi
+    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo pip install lit; fi
```

Comment:
Why can't we use `pip2` under Linux too?

Reply:
I don't know if it is aliased or not.